docs/input-filter.md

```diff
@@ -21,6 +21,7 @@ $_POST, $_GET and $_COOKIE arrays are only populated with stripped
 data.  In this simple example all I am doing is calling strip_tags() on
 the data.
 
+```
```

Comment:
Here we could have indicated a C block of code
```suggestion
```c
```

Reply:
Fix coming up... in the next commit...